src/shared/clients/ssmClient.ts

```diff
@@ -28,4 +36,29 @@ export class SsmClient {
         const response = await client.startSession({ Target: target })
         return response
     }
+
+    public async describeInstanceInformation(target: string): Promise<InstanceInformation> {
```

Comment:
it's good to avoid needless differences, but we can truncate verbose things when writing wrappers (which isn't quite the same as "changing for the sake of change"). "Information" adds nothing in most cases when it is on a function name.

```suggestion
    public async describeInstance(target: string): Promise<InstanceInformation> {
```